go-apps/meep-rnis/server/rnis.go

```diff
@@ -0,0 +1,694 @@
+	cellChangeSubscription1 := new(CellChangeSubscription1)
+
+	decoder := json.NewDecoder(r.Body)
+	err := decoder.Decode(&cellChangeSubscription1)
+	if err != nil {
+		log.Error(err.Error())
+		http.Error(w, err.Error(), http.StatusInternalServerError)
+		return
+	}
+
+	cellChangeSubscription := cellChangeSubscription1.CellChangeSubscription
+
+	selfUrl := strings.Split(cellChangeSubscription.Links.Self, "/")
+	subsIdStr := selfUrl[len(selfUrl)-1]
+
+	if subsIdStr != subIdParamStr {
+		http.Error(w, "Body content not matching parameter", http.StatusInternalServerError)
+		return
+	}
+
+	registerCc(cellChangeSubscription, subsIdStr)
+
+	_ = rc.JSONSetEntry(moduleRNIS+":"+cellChangeSubscriptionType+":"+subsIdStr, ".", convertCellChangeSubscriptionToJson(cellChangeSubscription))
+
+	response.CellChangeSubscription = cellChangeSubscription
+	jsonResponse, err := json.Marshal(response)
+	if err != nil {
+		log.Error(err.Error())
+		http.Error(w, err.Error(), http.StatusInternalServerError)
+		return
+	}
+	w.WriteHeader(http.StatusOK)
+	fmt.Fprintf(w, string(jsonResponse))
+}
+
+func delSubscription(keyPrefix string, subsId string) error {
+
+	err := rc.JSONDelEntry(keyPrefix+":"+subsId, ".")
+	deregisterCc(subsId)
+	return err
+}
+
+func cellChangeSubscriptionsDELETE(w http.ResponseWriter, r *http.Request) {
+	w.Header().Set("Content-Type", "application/json; charset=UTF-8")
+	vars := mux.Vars(r)
+
+	err := delSubscription(moduleRNIS+":"+cellChangeSubscriptionType, vars["subscriptionId"])
+	if err != nil {
+		http.Error(w, err.Error(), http.StatusInternalServerError)
+		return
+	}
+
+	w.WriteHeader(http.StatusNoContent)
+}
+
+func measRepUeReportSubscriptionsPUT(w http.ResponseWriter, r *http.Request) {
+	notImplemented(w, r)
+}
+
+func measRepUeReportSubscriptionsPOST(w http.ResponseWriter, r *http.Request) {
+	notImplemented(w, r)
+}
+
+func measRepUeReportSubscriptionsGET(w http.ResponseWriter, r *http.Request) {
+	notImplemented(w, r)
+}
+
+func measRepUeReportSubscriptionsDELETE(w http.ResponseWriter, r *http.Request) {
+	notImplemented(w, r)
+}
+
+func plmnInfoGET(w http.ResponseWriter, r *http.Request) {
+
+	w.Header().Set("Content-Type", "application/json; charset=UTF-8")
+
+	u, _ := url.Parse(r.URL.String())
+	log.Info("url: ", u.RequestURI())
+	q := u.Query()
+	appInsId := q.Get("app_ins_id")
+	appInsIdArray := strings.Split(appInsId, ",")
+
+	var response InlineResponse2001
+	atLeastOne := false
+
+	activeModel := sbi.GetActiveModel()
+	if activeModel == nil {
+		log.Error("No active scenarion")
+		w.WriteHeader(http.StatusNotFound)
+		return
+	}
+
+	//same for all plmnInfo
+	seconds := time.Now().Unix()
+	var timeStamp TimeStamp
+	timeStamp.Seconds = int32(seconds)
+
+	for _, meAppName := range appInsIdArray {
+		meAppName = strings.TrimSpace(meAppName)
+
+		n := activeModel.GetNode(meAppName)
+		if n == nil {
+			log.Error("Error node not found " + meAppName)
+			continue
+		}
+		meApp, ok := n.(*ceModel.Process)
+		if ok {
```

Comment:
To help reduce indentation:

if meApp, ok := n.(*ceModel.Process); ok {

}

Do this at each level.